.buildbot.sh

```diff
@@ -0,0 +1,17 @@
+#! /bin/sh
+
+set -e
+
+export CARGO_HOME="`pwd`/.cargo"
+export RUSTUP_HOME="`pwd`/.rustup"
+
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs > rustup.sh
+sh rustup.sh --default-host x86_64-unknown-linux-gnu --default-toolchain nightly -y --no-modify-path
+
+export PATH=`pwd`/.cargo/bin/:$PATH
+
+rustup component add --toolchain nightly rustfmt-preview || cargo +nightly install --force rustfmt-nightly
+
+cargo +nightly fmt --all -- --check gc_tests/tests/*
```

Comment:
You may have to jump through hoops if rustfmt is broken at the time of this running.

Laurie has a solution which skips rustfmt in that case, or you might be able to do something like mentioned here:
https://github.com/rust-lang/rustup/issues/2227#issuecomment-584754687